src/views/TradingCompetition/components/CompetitionHeadingText.tsx

```diff
@@ -21,8 +21,8 @@ const H2SizeStyles = (theme: DefaultTheme) => `
   }
 `
 
-const sharedStyles = (props: HeadingProps) => `
-  color: ${props.textColor ? props.textColor : '#ffffff'};
+const sharedStyles = (props) => `
+  color: ${props.textColor ? props.theme.colors[props.textColor] : '#ffffff'};
```

Comment:
I'm not sure any implementations of the `Heading1Text` / `Heading2Text` components even make use of this prop, I can probably rip this out. Will look tomorrow during trading comp code cleanup

Reply:
Please add the typing back unless Typescript can correctly infer the props.

Reply:
Done